docs/rules/no-div-regex.md

```diff
@@ -1,4 +1,4 @@
-# Disallow Regexs That Look Like Division (no-div-regex)
+# Disallow RegExps That Look Like Division (no-div-regex)
```

Comment:
To avoid contention around regexes vs regexen vs RegExps, should this just say "Regular Expressions"? There's no strong need to abbreviate here anyway.